tests/start-cypress-tests.sh

```diff
@@ -30,6 +30,16 @@ fi
 echo "Logging into Kube API server..."
 oc login --server=$CYPRESS_OC_CLUSTER_URL -u $CYPRESS_OC_CLUSTER_USER -p $CYPRESS_OC_CLUSTER_PASS --insecure-skip-tls-verify
 
+echo "Checking RedisGraph deployment."
+rgstatus=`oc get srcho searchoperator -o jsonpath="{.status.deployredisgraph}"`
```

Comment:
Need to add `-n open-cluster-management` here.

Reply:
Added. Thanks.